addon-test-support/ember-qunit/test-loader.js

```diff
@@ -57,5 +57,16 @@ export class TestLoader extends AbstractTestLoader {
    @method loadTests
  */
 export function loadTests() {
-  new TestLoader().loadModules();
+  if (!testLoaderInstance) {
+    getTestLoaderInstance();
+  }
+
+  return testLoaderInstance.loadModules();
 }
+
+export function getTestLoaderInstance() {
+  if (!testLoaderInstance) {
+    testLoaderInstance = new TestLoader();
```

Comment:
`testLoaderInstance` needs to be declared before assigning to it. Otherwise this is going to throw an error because modules are imported in strict mode.